test/api.js

```diff
@@ -33,13 +33,30 @@ const databases = {
       password: 'strapi',
     },
   },
+  mysql2: {
+    client: 'mysql2',
```

Comment:
Looks lke you are simply allowing mysql2 to be a client. contrary to sqlite if we do that why do we still need to manually try the packages & have an env var 🤔  isn't knex going to work with that directly ?